src/frontends/ir/src/utils.hpp

```diff
@@ -13,6 +14,10 @@ namespace ov {
 void operator>>(const std::stringstream& in, ov::element::Type& type);
 
 bool getStrAttribute(const pugi::xml_node& node, const std::string& name, std::string& value);
+Dimension strToDimension(const std::string& value);
+PartialShape strToPartialShape(const std::string& value);
+bool getDimensionFromAttribute(const pugi::xml_node& node, const std::string& name, Dimension& value);
+bool getPartialShapeFromAttribute(const pugi::xml_node& node, const std::string& name, PartialShape& value);
```

Comment:
Please use snake_case namings for new functions.

Reply:
Done.